src/pharmpy/tools/covsearch/tool.py

```diff
@@ -24,6 +24,7 @@
 )
 from pharmpy.tools.mfl.parse import parse as mfl_parse
 from pharmpy.tools.modelfit import create_fit_workflow
+from pharmpy.tools.run import _is_strictness_fulfilled
```

Comment:
A private function should not be imported in other modules, but I think it would make sense to make this function public and also exported in the `tools`-module.